turbopack/crates/turbopack-browser/src/chunking_context.rs

```diff
@@ -629,6 +637,11 @@ impl ChunkingContext for BrowserChunkingContext {
         self.minify_type.cell()
     }
 
+    #[turbo_tasks::function]
+    fn should_inline_css(&self) -> Vc<bool> {
```

Comment:
This should really be something like
```suggestion
    fn should_use_absolute_url_references(&self) -> Vc<bool> {
```